### ISSUE TITLE
info: TypeError: null is not an object (evaluating 'child.transform')

### ISSUE THREAD

**Issue by niknickolas** [amir20/phantomjs-node#424]
### Environment

- OS: Ubuntu 14
- Node version (`node -v`): v5.9.1
- package version: '2.0.10'

This  error occurs when I launch my code on server side and when I use method evaluate, after this output I  stopped execution because on local computer it is working very well. Environment of local server is similar to production server.
 
info: TypeError: null is not an object (evaluating 'child.transform')
info: 
info:   phantomjs://code/shim.js:131 in transform
info:   phantomjs://code/shim.js:112 in read

What I have to do to get same  result  as on my local server ?

**Comment by amir20** [comment]
I have no idea. There could be a lot of things going on. I am not sure what kind of help I can provide. Make sure phantomjs is installed at least. I have never seen this error.

**Comment by niknickolas** [comment]
Hello.

I wrote TaskRunner to download files from sites that runs under nodejs.
NodeJS gets a new job from a beanstalk queue and creates a new instance of
Phantom . I can create as many instances of Phantom how much is available
RAM on the server, they will all run in parallel for a given scenario.
On the local server I all fully tested, as the primary server this is not
possible, there should be only debugged the code because any delay in work
will lead to the accumulation of tasks and the desired result is immediate.
And here are all the achievements in tesota version failed on the eyes ))))

2016-03-25 19:14 GMT+02:00 Amir Raminfar <notifications@github.com>:

**Comment by amir20** [comment]
Closing. Out of scope for this repo to resolve.

**Comment by MichaelJCole** [comment]
Hey @amir20 I think I got the same error with the following code:

```
page.setting('userAgent', undefined)
```

**Comment by vinnitu** [comment]
I got this error with using proxy
